docs/content/contribution-guidelines/compatibility-e2e.md

```diff
@@ -0,0 +1,38 @@
+# KubeStellar & KubeFlex Compatibility E2E Testing
+
+## Overview
+KubeStellar runs automated end-to-end (E2E) compatibility tests with KubeFlex on every main branch merge and release for either project. This ensures integration issues are detected early.
+
+## How It Works
+- **Trigger:** The test is triggered by a Prow job (`pull-kubestellar-kubeflex-compatibility-e2e`) on merges/releases to KubeStellar or KubeFlex. Postsubmit jobs for both repositories are defined in the `kubestellar/infra` Prow configuration.
+- **Cluster:** Tests run in an Oracle Cloud Infrastructure (OCI) Kubernetes cluster.
+- **Matrix:** The job tests multiple version pairs (main/main, main/stable, stable/main, stable/stable).
+- **Periodic:** A periodic job runs nightly to ensure regular compatibility coverage even without active merges.
+- **Script:** The E2E script installs both projects and validates integration scenarios:
+  - **WEC registration** – registers a simulated Workload Execution Cluster and verifies it appears in KubeStellar.
```

Comment:
In website source files you need four spaces of indenting to indicate a nested paragraph.